src/api/app/helpers/maintenance_helper.rb

```diff
@@ -237,9 +236,8 @@ def copy_single_binary(arch, target_repository, source_project_name, source_pack
     cp_params[:setupdateinfoid] = update_info_id if update_info_id
     cp_params[:setrelease] = setrelease if setrelease
     cp_params[:multibuild] = '1' unless source_package_name.include?(':')
-    # rubocop:disable Layout/LineLength
     cp_path = "/build/#{CGI.escape(target_repository.project.name)}/#{URI.escape(target_repository.name)}/#{URI.escape(arch.name)}/#{URI.escape(target_package_name)}"
-    # rubocop:enable Layout/LineLength
+
```

Comment:
This empty line is really needed?

Reply:
I'll leave it. I don't have a strong opinion on this one. It was corrected like this. Fine for you?

Reply:
Fine for me :slightly_smiling_face: